reviews/sandbox/2023/2023-telepresence-annual.md

```diff
@@ -0,0 +1,91 @@
+# Telepresence Sandbox Annual Review
+
+The goal of the Telepresence project is to accelerate the "developer inner loop" for cloud-native application development on Kubernetes. Telepresence achieves this by creating a dynamic network bridge between a local development environment (e.g., a laptop) and a remote Kubernetes cluster.
+
+Telepresence was presented to the TOC on April 2018. Please see the project's [Sandbox proposal](https://github.com/cncf/toc/blob/master/proposals/sandbox/telepresence.adoc).
+
+## Annual Review Questions
+
+### Metrics
+
+| Metric       |  Pre-Sandbox |     2020     |    2021      |   2022    |   2023   |
+| ------------ | ------------ | ------------ | ------------ | --------- | -------- |
+| Docker Pulls | 100K+        | 10M+         | 10M+         | 10M+      | 10M+     |
+| Stars        | 700+         | 2700+        | 4000         | 5.1k      | 5.8K     |
+| Forks        | ~50          | ~190         | ~300         | 400+      | 400+     |
+
+
+The CNCF provides Telepresence Github metrics through its [devstats site](https://telepresence.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m).
+
+### Maintainers
+
+There are eight maintainers total. Of these, four commit to the project on a daily basis and are employed by Ambassador Labs.
```

Comment:
Only six maintainers are listed in https://github.com/telepresenceio/telepresence/blob/release/v2/MAINTAINERS.md where does the difference come from?